sos/plugins/ds.py

```diff
@@ -43,7 +43,7 @@ def setup(self):
         self.add_forbidden_path("/etc/dirsrv/slapd*/key3.db")
         self.add_forbidden_path("/etc/dirsrv/slapd*/pwfile.txt")
         self.add_forbidden_path("/etc/dirsrv/slapd*/*passw*")
-        self.add_forbidden_path("/etc/dirsrv/admin-serv/key3.db")
+        self.add_forbidden_path("/etc/dirsrv/admin-serv/key[3-4].db")
```

Comment:
I think for a range of values that are just two adjacent values it is clearer to write `key[34].db`, but maybe that's just my personal taste (I had to check Py regexes supported the range notation!).

Anyway, just a comment - not anything to hold this change up for.